flashinfer/fused_moe/cute_dsl/tuner.py

```
@@ -180,7 +184,8 @@ def get_moe_valid_tactics() -> List[Tuple]:
 
 # Pre-generate all valid tactics
 # tile_size=128: 2 GEMM1 tactics × 4 GEMM2 tactics = 8
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)